.github/workflows/releases.yaml

```diff
@@ -322,8 +322,15 @@ jobs:
       - name: Build taplo-full
         run: |
           cargo build --verbose --release --bin taplo --target ${{ matrix.triple }} --features toml-test,lsp
-          gzip -c ./target/${{ matrix.triple }}/release/taplo.exe > ./taplo-full-windows-${{ matrix.platform }}.gz
+          # zip
           Compress-Archive -Path ./target/${{ matrix.triple }}/release/taplo.exe -DestinationPath ./taplo-full-windows-${{ matrix.platform }}.zip
+          # gzip
+          $file = [System.IO.File]::Open('.\target\${{ matrix.triple }}\release\taplo.exe', [System.IO.FileMode]::Open)
+          $archive = [System.IO.File]::Create('.\taplo-full-windows-${{ matrix.platform }}.gz')
+          $compressor = [System.IO.Compression.GZipStream]::new($archive, [System.IO.Compression.CompressionMode]::Compress)
+          $file.CopyTo($compressor)
+          Start-Sleep -Seconds 2
+          $compressor.close()
```

Comment:
Sleep is because I got an error once or twice when testing it, although it should work without it.

> Looking a bit on internet I suspect that "disposing" the stream rather than closing it might be the solution.

`Dispose()` calls `Close()` so it doesn't matter

Also we've been using this logic for a long time now
https://github.com/lapce/lapce/blob/0d4f6fd30a4e463a2d617fe188a24a55e73b62a3/.github/workflows/release.yml#L96-L101

Reply:
This looks brittle to me (relying on time instead of events) but maybe PowerShell cannot do better. I'm not competent in that domain to argue.